sql/changes/1.8/constrain-transaction-tables.sql.checks.pl

```diff
@@ -352,11 +352,11 @@ package _18_upgrade_checks;
 the database anymore. As a result, the files attached to these transactions
 cannot remain connected. Two options are available:
 
-1. 'Delete' the files listed below  
+1. 'Delete' the files listed below
    This action can't be undone. As the files aren't available in the
    application, due to the fact that they are attached to orphaned
    transactions, this may not make a difference.
-2. 'Copy' the files to the 'incoming' queue  
```

Comment:
(same here)

Reply:
Fixed, using trailing spaces as markdown is not ideal.